pom.xml

```diff
@@ -65,9 +65,9 @@
   </scm>
 
   <properties>
-    <revision>2.4.2</revision>
+    <revision>2.5.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.164.3</jenkins.version>
```

Comment:
FWIW usually preferable to update to the oldest LTS line that has whatever core API you need. Especially in this case, because the plugin is an API intended to be used by other plugins, requiring the second-newest LTS release means that any plugin that wants to pick up the latest version of this plugin also requires the that release, which is a bit prohibitive.

Reply:
In this case `2.107.3` was the minimum required baseline as far as I know.